lib/rules/i18n-no-template-literal.js

```diff
@@ -0,0 +1,77 @@
+'use strict';
+
+const { I18N_METHODS } = require('../i18n.js');
+
+module.exports = {
+  meta: {
+    docs: {
+      category: 'Possible Errors',
+      description:
+        'Ensure predictable static values are passed as i18n method arguments',
+      recommended: false,
+    },
+    fixable: 'code',
+    schema: [],
+    messages: {
+      noStaticTemplate:
+        'The first argument of i18n.{{method}}() must be a static string.',
+    },
+  },
+
+  create(context) {
+    return {
+      CallExpression(node) {
+        const { callee, arguments: args } = node;
+
+        if (!callee.type === 'MemberExpression' || !args.length) {
+          return;
+        }
+
+        const { object, property } = node.callee;
+
+        if (!object || !object.name || object.name !== 'i18n') {
```

Comment:
nit: I suppose we could use the optional chaining operator these days:

```suggestion
        if (object?.name !== 'i18n') {
```

Reply:
Done.